website/docs/timeline.md

```diff
@@ -3,40 +3,384 @@ title: Timeline
+- **REPLACECOMMIT**: This denotes an action that adds new data files to replace existing files in a table.
+- **CLEAN**: This table service action eliminates older versions of files in the table that is no longer needed.
+  - Hudi automatically supports [clean](https://hudi.apache.org/docs/basic_configurations#Clean-Configs) as the default behavior. 
+  - However, the user can disable this setting and perform the clean separately. You can configure the clean through Spark SQL call procedure, Scala within a Spark Shell and the Hudi CLI tool. 
+- **DELTACOMMIT**: This refers to an atomic write of a batch of records into a Merge-On-Read type table, where some/all of the data could be just written to delta logs.
+  - [Deltastreamer](https://hudi.apache.org/docs/hoodie_deltastreamer) and Spark DataSource automatically support this; you don't need to set configs.
+- **COMPACTION**: This is background activity to reconcile differential data structures within Hudi. For example, moving updates from row-based log files to columnar formats. Internally, compaction manifests as a unique commit on the timeline.
+  - Hudi automatically supports compaction as the default behavior. 
+  - However, you can disable this setting and perform the compactions separately. You can configure the [compactions](https://hudi.apache.org/docs/basic_configurations#Compaction-Configs) through the Spark SQL call procedure, Scala within a Spark Shell and the Hudi CLI tool. 
+- **ROLLBACK**: This indicates that a commit/delta commit was unsuccessful & rolled back, removing any partial files produced during the writes.
+  - Hudi automatically supports rollbacks as the default behavior. 
+  - However, the user can disable this setting and perform rollbacks separately. You can configure the [rollbacks](https://hudi.apache.org/docs/configurations#hoodierollbackparallelism) through the Spark SQL call procedure, Scala within a Spark Shell and the Hudi CLI tool. 
+- **SAVEPOINT**: This marks certain file groups as "saved," where the cleaner will not delete them. It helps restore the table to a point on the timeline for disaster/data recovery scenarios.
+  - This is supported by the Spark SQL call procedure, Scala within a Spark Shell and the Hudi CLI tool. You manually set up this action. 
+- **RESTORE**: This indicates an action to restore the table to a previous state.
+  - This is supported by the Spark SQL call procedure, Scala within a Spark Shell and the Hudi CLI tool. You manually set up this action. 
+- **INDEXING**: This denotes an action to [asynchronounsly build the index](https://hudi.apache.org/docs/metadata_indexing) in the metadata table.
+
+## States
+A state denotes if an action has been scheduled, in progress or completed. There are three different states an action can be in: 
+- **REQUESTED**: Denotes an action has been scheduled, but has not initiated.
+- **INFLIGHT**: Denotes an action is currently being performed.
```

Comment:
`has been scheduled, but has not initiated.` -> ` has initiated, but has not been scheduled.`

Reply:
fixed- thanks!